tests/Unit/IO/H5/Python/Test_ExtractDatFromH5.py

```diff
@@ -44,7 +44,7 @@ def test_extract_dat_files(self):
         h5file.close()
 
         # All defaults, data should be same as expected
-        extract_dat_files(self.h5_filename, out_dir=self.created_dir)
+        extract_dat_files(self.h5_filename, self.created_dir, 1)
```

Comment:
Always use named arguments in Python when it improves clarity. Here it does, so write `out_dir=` and `num_cores=`.